takari-builder-apt/src/main/java/io/takari/builder/apt/MojoGenerationVisitor.java

```diff
@@ -90,14 +107,11 @@ private void checkResolutionScope(io.takari.builder.ResolutionScope scope) {
   public void visitBuilder(BuilderMethod metadata) {
     Builder ann = metadata.annotation();
     String defaultPhase = ann.defaultPhase().name();
-    String resolutionScope =
-        this.resolutionScope != null ? this.resolutionScope : ResolutionScope.NONE.name();
 
     AnnotationSpec.Builder annBuilder = AnnotationSpec.builder(Mojo.class) //
         .addMember("name", "$S", ann.name()) //
         .addMember("defaultPhase", "$T.$L", LifecyclePhase.class, defaultPhase) //
         .addMember("requiresProject", "$L", true) //
-        .addMember("requiresDependencyResolution", "$T.$L", ResolutionScope.class, resolutionScope) //
```

Comment:
I think we want `ResolutionScope.NONE` here, unless you are certain this is the default.

Reply:
I was pretty sure that was the default, but I will have to look at the code again... or just add the NONE scope here.